api/db/repositories/orderHelpers/orderHelpers.js

```diff
@@ -0,0 +1,99 @@
+const orderDomain = require('@domains/ordersDomain')
+const orderRepo = require('@repos/orderRepository')
+const productRepo = require('@repos/productRepository')
+const { Verification, Resources } = require('@verify/verification')
+
+class OrderHelpers {
+    static async AddCounters(order, productId, quantity) {
+        const product = await productRepo.getById(productId)
+        
+        const orderDataValues = order.dataValues
+        let productsCount = orderDataValues.productsCount
+        let productsTotalPrice = orderDataValues.productsTotalPrice
+
+        switch (order.currency) {
+            case 'pln':
+                productsTotalPrice += product.price.pln * quantity
+                break
+
+            case 'eur':
+                productsTotalPrice += product.price.eur * quantity
+                break
+
+            case 'usd':
+                productsTotalPrice += product.price.usd * quantity
+                break
+        }
+
+        productsCount += quantity
+
+        orderDataValues.productsCount = productsCount
```

Comment:
you can get rid of `productsCount` variable and just do `orderDataValues.productsCount += quantity `